core.cpp

```diff
@@ -125,9 +130,10 @@ static void io_latch()
 		break;
 	case OT_DIRECT_POS:
 	case OT_DIRECT_NEG:
+	case OT_GREEN_IQ:
 		for (int i = 0; i <= NUM_ZONES; i++)
 		{
-			if (eot == OT_DIRECT_POS)
+			if (eot == OT_DIRECT_POS || eot == OT_GREEN_IQ)
```

Comment:
If direct positive works with your board, please just use direct positive and add documentation stating that to the wiki.